service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsController.java

```diff
@@ -237,8 +240,17 @@ public void handleSubmittedEvent(@RequestBody CallbackRequest callbackRequest) {
             }
 
             hearingsService.findHearingBooking(caseData.getSelectedHearingId(), caseData.getHearingDetails())
-                .filter(hearing -> isNotEmpty(hearing.getNoticeOfHearing()))
-                .ifPresent(hearing -> publishEvent(new SendNoticeOfHearing(caseData, hearing)));
+                .ifPresent(hearingBooking -> {
+                    if (isNotEmpty(hearingBooking.getNoticeOfHearing())) {
+                        publishEvent(new SendNoticeOfHearing(caseData, hearingBooking));
+                    }
+
+                    if (isNewOrReListedHearing(caseData)
+                        && hasTemporaryJudgeEmail(hearingBooking)
```

Comment:
can just check with hearingBooking.getHearingJudgeLabel, this label is only populated if temporary judge is being used for hearing, so can just do null check on this. Also email is mandatory/always present so don't think it needs checking again